### PR TITLE
Add new parameter forced_decoder_ids to OpenAIWhisperParserLocal + small bug fix

### PR DESCRIPTION
**Pull request by idcore**
- Description: new parameter forced_decoder_ids for OpenAIWhisperParserLocal to force input language, and enable optional translate mode. Usage example:
    processor = WhisperProcessor.from_pretrained("openai/whisper-medium")  
    forced_decoder_ids = processor.get_decoder_prompt_ids(language="french", task="transcribe") 
    #forced_decoder_ids = processor.get_decoder_prompt_ids(language="french", task="translate") 
    loader = GenericLoader(YoutubeAudioLoader(urls, save_dir), OpenAIWhisperParserLocal(lang_model="openai/whisper-medium",forced_decoder_ids=forced_decoder_ids))
  - Issue #8792
  - Tag maintainer: @rlancemartin, @eyurtsev

Please make sure you're PR is passing linting and testing before submitting. Run `make format`, `make lint` and `make test` to check this locally.

See contribution guidelines for more information on how to write/run tests, lint, etc: https://github.com/hwchase17/langchain/blob/master/.github/CONTRIBUTING.md
 -->

### CODE REVIEW

File: libs/langchain/langchain/document_loaders/parsers/audio.py [hunk 1]
```diff
@@ -136,10 +156,19 @@ def __init__(self, device: str = "0", lang_model: Optional[str] = None):
         # load model for inference
         self.pipe = pipeline(
             "automatic-speech-recognition",
-            model="openai/whisper-medium",
+            model=self.lang_model,  # fix to use model name that was evaluated earlier
```
Comment (hwchase17) [inline libs/langchain/langchain/document_loaders/parsers/audio.py:159]: i would remove this comment, it loses meaning outside of this pr

File: libs/langchain/langchain/document_loaders/parsers/audio.py [hunk 2]
```diff
@@ -136,10 +156,19 @@ def __init__(self, device: str = "0", lang_model: Optional[str] = None):
...
             chunk_length_s=30,
             device=self.device,
         )
+        try:
+            if forced_decoder_ids is not None:
```
Comment (hwchase17) [inline libs/langchain/langchain/document_loaders/parsers/audio.py:164]: nit: i feel like its slightly nicer to do:
```
if ...:
    try:
    ...
    except:
    ...
```
dont care super strongly tho

File: libs/langchain/langchain/document_loaders/parsers/audio.py (hunk 1 above)
Comment (idcore) [reply libs/langchain/langchain/document_loaders/parsers/audio.py]: Thank you. Fixed.

File: libs/langchain/langchain/document_loaders/parsers/audio.py (hunk 2 above)
Comment (idcore) [reply libs/langchain/langchain/document_loaders/parsers/audio.py]: Thank you. Fixed.

**Review by hwchase17 — APPROVED** [review]
lgtm! thanks

**Comment by idcore** [comment]
Minor fix - shortened 1 line due to lint failed check. Will look into why local lint didn't displayed it for me.

**Pull request merged**